google/resource_compute_instance_group_manager_test.go

```diff
@@ -160,12 +160,12 @@ func TestAccInstanceGroupManager_rollingUpdatePolicy(t *testing.T) {
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckInstanceGroupManagerDestroy,
 		Steps: []resource.TestStep{
-			resource.TestStep{
```

Comment:
I think this line needs to be added back in

Reply:
nah you don't need the type there, just `{` is sufficient since the type is defined in the array definition the line above